src/OpenSearch.Client/QueryDsl/Abstractions/Container/QueryContainer-Dsl.cs

```diff
@@ -131,5 +131,42 @@ public void Accept(IQueryVisitor visitor)
 
 		// ReSharper disable once UnusedMember.Global
 		internal bool ShouldSerialize(IJsonFormatterResolver formatterResolver) => IsWritable;
+
+
+		public QueryContainer Name(string name)
+		{
+			ContainedQuery.Name = name;
+			return this;
+		}
+
+		/// <summary>
+		/// Applies or removes the `strict` attribute to the query container and all child sub-queries.
+		/// </summary>
+		/// <param name="strict">When null, the value is taken from the root query.</param>
+		/// <returns></returns>
+		public QueryContainer Strict(bool? strict = true)
```

Comment:
I'd prefer if this was something like `Strict(bool strict = true, bool recurse = false)` so that the recurse is something opted into explicitly, and otherwise behaves the same as all other strict setters.